.github/workflows/pr-check.yml

```diff
@@ -57,7 +56,6 @@ jobs:
 
   Backend:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'push' || !github.event.pull_request.draft }}
```

Comment:
We don't need those git event checking anymore?

Reply:
I discovered that github actions are actually unlimited for public repos 😅. I thought they were still subject to the 3000 min limit hence #24

Reply:
yes, they are unlimited, just remove that